frappe/integrations/doctype/google_calendar/google_calendar.py

```diff
@@ -47,10 +50,10 @@
 }
 
 framework_frequencies = {
-	"Daily": "RRULE:FREQ=DAILY;",
-	"Weekly": "RRULE:FREQ=WEEKLY;",
-	"Monthly": "RRULE:FREQ=MONTHLY;",
-	"Yearly": "RRULE:FREQ=YEARLY;",
+	"Daily": {"FREQ": "DAILY"},
```

Comment:
Not sure what these changes entail - will have to revisit this once I get the hang of calendar jargon and all that jazz (like https://icalendar.org/iCalendar-RFC-5545/3-8-5-3-recurrence-rule.html)